kornia/augmentation/random_generator/_3d/affine.py

```diff
@@ -4,8 +4,8 @@
 from torch.distributions import Uniform
 
 from kornia.augmentation.random_generator.base import RandomGeneratorBase
-from kornia.augmentation.utils import _adapted_rsampling, _adapted_uniform, _singular_range_check, _tuple_range_reader
-from kornia.utils.helpers import _deprecated, _extract_device_dtype
+from kornia.augmentation.utils import _adapted_rsampling, _singular_range_check, _tuple_range_reader
+from kornia.utils.helpers import _extract_device_dtype
 
 
 class AffineGenerator3D(RandomGeneratorBase):
```

Comment:
Shouldn't these classes have "Random" in their name?

Reply:
Possibly